### PR TITLE
21134 Added continued in entity types to business searches

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21134

*Description of changes:*
- app version = 5.10.21
- imported latest Business Lookup shared component
- added C, CBEN, CCC and CUL to amalgamation business search
- renamed getters and session storage key to better match environment key names

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "@bcrs-shared-components/approval-type": "1.0.19",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
-    "@bcrs-shared-components/business-lookup": "1.3.3",
+    "@bcrs-shared-components/business-lookup": "1.3.4",
```
Comment (severinbeauvais) [inline package.json:20]: In this updated component, the default entity types peop now includes C, CBEN, CCC and CUL. (Some business searches use the default; other request specific types only.)

**Comment by severinbeauvais** [comment]
In this UI, there are 3 places that Business Search is used:
- firm business search (uses updated default types from shared component)
- amalgamation business search (added C, CBEN. CCC and CUL)
- continuation in extrapro business search (no change needed -- searches for A only)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-702-8de2bfzl.web.app

**Review by JazzarKarim — APPROVED** [review]
👍

**Comment by ketaki-deodhar** [comment]
a bit late to this but looked through the PR. Looks good

**Pull request merged**